crates/storage-query-datafusion/src/lib.rs

```diff
@@ -20,6 +20,7 @@ mod invocation_state;
 mod invocation_status;
 mod journal;
 mod keyed_service_status;
+mod node;
```

Comment:
Not a blocker and certainly can be a follow-up on top of the PR stack:
It feels like we should split user table and cluster table into two crates, then storage-query-datafusion becomes the core/common crate. Do you think this is possible?

Reply:
I was thinking about that as well. It is definitely possible with some refactoring. I will make sure to follow up in this.